zerver/lib/actions.py

```diff
@@ -592,6 +593,8 @@ def do_set_realm_property(realm: Realm, name: str, value: Any) -> None:
         'Cannot update %s: %s is not an instance of %s' % (
             name, value, property_type,))
 
+    if name == "email_address_visibility":
+        old_value = getattr(realm, name)
```

Comment:
Let's just store the `old_value` unconditionally if we're doing this; feels more readable.